### ISSUE TITLE
[DOCS] <Add URL types to TypeScript>

### ISSUE THREAD

**Issue by TheOtherBrian1** [linvo-io/linvo-scraper#21]
### Description

I have been trying to use Linvo's scraping library found in the [linkedin.global.page.service.ts ](https://github.com/linvo-io/linvo-scraper/blob/main/lib/linkedin/linkedin.global.page.service.ts). Unfortunately, in the required parameters section it doesn't specify which URLs are acceptable for scraping.

```javascript
interface RequiredData {
  page: number;
  url: string; //WHAT DOES THE URL LOOK LIKE
  remove_overlapping: boolean;
}
```
It would be very helpful to provide one or more of the following:

- TypeScript URL checking
- A descriptive URL related comment in the linkedin.global.page.service.ts file
- A descriptive error message that outlines what webpages are appropriate
- Documentation that specifies which webpages can be scraped

Right now, when I provide the following URL as an argument 
> https://www.linkedin.com/search/results/all/?keywords=DATA%20SCIENCE%20JOBS&origin=GLOBAL_SEARCH_HEADER&sid=Q~M
I receive the warning: 
> originalMessage: 'Cannot navigate to invalid URL'
Unfortunately, for less experienced users like myself, this error message is not descriptive enough to be actionable.

**Comment by nevo-david** [comment]
Yes, it has to be with "https://www.linkedin.com/search/results/people/" can you add it to the docs?

**Comment by TheOtherBrian1** [comment]
Yeah, I can add it

**Comment by nevo-david** [comment]
Awsome! Assigning it to you!
